### DISCUSSION TITLE
What services to report for each provider

### DISCUSSION THREAD

**Discussion by ryan-thomas-peterson** [CMSgov/price-transparency-guide#64] [Q&A] [answered]
Right now, the implementation of how to select what services should be reported for each provider appears left up to the end user to determine.  Understanding the scope of what a provider can bill vs. the much narrower scope of what they actually do can be a challenge.  

An end user creating the in network rate files could reasonably take the stance that they will only report on observed services that they have seen billed.  Another might create a standard set of services that all specialists sharing an NUCC taxonomy could do and report on those.  Yet another, could require providers to provide a listing of every code each provider performs.  Or leverage Medicare's Provider Utilization Files (though those are a couple years old).  

Our current designs historical use with specialty to captures services we've seen performed and services that might reasonably be performed by a person with the same specialty (again based on our own historical data).  

Different implementation decisions may lead to vastly different numbers of services being reported and wanted to call in case there was any more specific guidance.

**Comment by ghost** [comment, marked as answer]
I disagree, historical use is implied in the Allowed-Amounts file which is based on claim history. The In-Network Negotiated Rates file would be the complete "book" of services for a provider within a plan for their negotiated rates seen or not. The only limitation of a published amount, in my opinion, would be based on taxonomy/specialty of the provider and the associated billing code to eliminate trying to attain a negotiated rate for a procedure that would not feasible (or if adjudicated would return an error or zero amount).

The file description in the readme.MD states for the In-Network-Rates file (which is a Cliff notes of the regulation) and my bold 
_"In-Network Negotiated Rates File Under the finalized rules, a plan or issuer must disclose in-network provider negotiated rates **for all items and services** through a machine-readable file."_

Also found in the regulation at pg 76 at https://www.cms.gov/CCIIO/Resources/Regulations-and-Guidance/Downloads/CMS-Transparency-in-Coverage-9915F.pdf 
_"To clarify, the final rules define in-network provider to mean any provider of items and services with which the plan or issuer, or a third-party for a plan or issuer, has a contract setting forth the terms under which a covered item or service may be provided to a participant, 
beneficiary, or enrollee."_

If you have a contract with a provider to reimburse them for "a covered item or service" you appear to be required to provide that information regardless if the provider has historically billed for such "covered item or service". 

Again my opinion, that is why the In-Network Negotiated Rates file is the most challenging one of the 3 MRFs.

**Comment by tdfow** [comment]
@BobSyracuse, yes, the In-Network File is the most challenging!  I am not seeing any guidance on the handling of the multitude of items and services.   
It doesn't make sense that the file should report the NPI info for provider's that are in the network for the negotiated rates of an  item/services, if outside their license/specialty for the item/service.

**Reply by ghost** [reply]
@tdfow Yes the guidance as a whole is lacking to the granularity needed for design/development to support the mandate as seen by the questions being posted. One aspect being the concerns about the negotiated rates being generated that do not cover the full "book" when it is outside their license/specialty for the item/service. Unfortunately, that still leaves a lot of items/services that are contracted the provider may have never billed for but are still within their license/specialty we have to report on that are the current negotiated rate for the reporting month. The CMS team here seems to be answering the questions as they can when asked. But it would have better to have a commentary/review of not just the written rules/mandates, but the same level of commentary/review for the technical aspect/schemas to flush out these questions/details prior to assigning a 01/01/22 date for the generation of the first set of files.

**Reply by ryan-thomas-peterson** [reply]
Totally agree on the aggressive timing, @BobSyracuse.   And, even with this layer of filtering (which seems to make sense in an attempt to somewhat limit file sizes), the continual maintenance of which standardized codes map to which license / specialty is left to each individual implementer.  A great standardized resource starting point that might be provided could be a listing of what CPT / HCPCS services fall in scope of each taxonomy.

**Comment by tdfow** [comment]
#28 @ryan-thomas-peterson

**Comment by jonberkery-work** [comment]
To @ryan-thomas-peterson's suggestion of - "A great standardized resource starting point that might be provided could be a listing of what CPT / HCPCS services fall in scope of each taxonomy."

Due to the massive file size of the in-network-rates file, the discrepancy between interpretations of the billing codes that each provider type can be reimbursed for, and consumer transfer size limitations (governed by the consumer's individual ISP) I would agree with this recommendation.

I would like to propose a solution to this audience....

Would it be beneficial to limit the in-network-rate file and the allowed-amounts file to the "500 procedure" codes that are expected in the first implementation of the "internet based self-service tool."  The "500 procedures" are identified in Table 1 of the [CMS-Transparency-in-Coverage-9915](https://www.cms.gov/CCIIO/Resources/Regulations-and-Guidance/Downloads/CMS-Transparency-in-Coverage-9915F.pdf) Documentation.

In my opinion this will
1. Reduce many of the "file size obstacles" file builders and file consumers are encountering
2. Align these files with the requirements identified for Phase 1 of the internet based self-service tool referenced in [CMS-Transparency-in-Coverage-9915](https://www.cms.gov/CCIIO/Resources/Regulations-and-Guidance/Downloads/CMS-Transparency-in-Coverage-9915F.pdf) Documentation.
3. Improve the consistency of the files being generated by many different and diverse players impacted by this rule.
4. Allow for both file builders and file consumers to make more informed recommendations to improve/adjust these files in future iterations.

I am open to feedback on this solution.

@BobSyracuse 
@tdfow 
@shaselton-usds 
@shaselton

**Reply by ghost** [reply]
@jonberkery-work @shaselton I would support this based on the 4 points you listed as it aligns with the "500 procedures" and allows for a foundation to work from to allow the list to mature based on feedback.

**Reply by rakeshkatte** [reply]
@jonberkery-work I really like this idea, it sure does make a lot of sense. 
The point that @ryan-thomas-peterson brought up about the mapping between taxonomy and the codes that need to be reported under it, would help build a more efficient data set. Is there such a mapping available anywhere today?

**Reply by jonberkery-work** [reply]
Thanks for the feedback @BobSyracuse @rakeshkatte.

@rakeshkatte, If you are looking for a mapping between the valid "billing codes" to each provider taxonomy code, I am not aware of one.  I agree that would also be helpful in defining scope of this file.

The one flaw in this approach is the terminology of the rule.  I don't see where the rule specifically references the 500 codes when talking about the MRFs, I only see where it says "all" ..... not sure if that means all 500, or all 18,000.  Would love some guidance.

@tdfow
@shaselton 
@shaselton-usds

**Reply by ryan-thomas-peterson** [reply]
New support using the catch-all billing code can help in some scenarios like % of billed contracts, but will not in others.  This remains an open question.

**Comment by shaselton-usds** [comment]
Wrapping some old threads here -- 

The final rules for the machine-readable files require disclosure for all covered items and services.  The 500 items and services only applies to the self-service tool.

@BobSyracuse is correct in his answer for his interruption for the in-network file requirements (which is why I'm flagging his answer as the 'correct' one)

**Reply by Paul-Sousa** [reply]
@shaselton-usds just to confirm, should we include a negotiated rate regardless of a plan’s benefits?  Does a “covered” item/service imply it is covered at the benefit level, or that it is an item/service we have a contracted rate with the provider for?
